src/couch/include/couch_db.hrl

```diff
@@ -40,6 +40,8 @@
     <<"_users">>
 ]).
 
+-define(DOCUMENT_SIZE_LIMIT, 8000000).
```

Comment:
Please use units in name of constants (i.e. `DOCUMENT_SIZE_LIMIT_BYTES`).

Reply:
Added. See https://github.com/apache/couchdb/pull/3632/commits/64d4741bce0fe9b9fed2d5b2a3bf6816ff7a1073.